DevTools/UserspaceEmulator/Emulator.cpp

```diff
@@ -648,9 +648,9 @@ int Emulator::virt$select(FlatPtr params_addr)
     Syscall::SC_select_params params;
     mmu().copy_from_vm(&params, params_addr, sizeof(params));
 
-    fd_set readfds;
-    fd_set writefds;
-    fd_set exceptfds;
+    fd_set readfds {};
+    fd_set writefds {};
+    fd_set exceptfds {};
```

Comment:
`fd_set` is a C type and should be initialized with `FD_ZERO()` or something.

And correct me if I'm wrong, but does adding an explicit `{}` even change anything? I believe the default constructor is invoked anyway.

Reply:
`{}` is aggregate initialization, and with no default constructor present, it will zero-initialize primitive members.
(At least that's how I understand it. :)